llvm/lib/Target/RISCV/RISCVISelDAGToDAG.cpp

```diff
@@ -1228,6 +1228,10 @@ void RISCVDAGToDAGISel::Select(SDNode *Node) {
       if (!Subtarget->hasVendorXTHeadBb())
         return false;
 
+      if ((Subtarget->hasStdExtZba() || Subtarget->hasStdExtZbb()) &&
```

Comment:
Should we be checking bit 31 for hasStdExtZba and bit 7 and 15 for hasStdExtZbb?

Reply:
Done.